feedback/githubService.js

```diff
@@ -0,0 +1,23 @@
+export async function createGitHubIssue(label, title, body) {
+  const url = `https://api.github.com/repos/${process.env.REPO_OWNER}/${process.env.REPO_NAME}/issues`
+  const headers = {
+    Authorization: `token ${process.env.GITHUB_TOKEN}`,
+    Accept: 'application/vnd.github.v3+json',
+    'Content-Type': 'application/json'
+  }
+
+  const response = await fetch(url, {
+    method: 'POST',
+    headers,
+    body: JSON.stringify({
+      title,
+      body,
+      labels: [label]
+    })
+  })
+
+  if (!response.ok) {
+    const error = await response.json()
+    throw new Error(`GitHub API error: ${error.message}`)
```

Comment:
Initially I did not trust this.  However, I messed with it a few different ways and GitHub did make errors that made sense and those bubbled to the client nicely.

Reply:
I was rewriting them all originally, but I found the same. Also, since we only use it within this module, we will always send it what we know to be properly formatted.